source/sass/patterns/organisms/content-header.scss

```diff
@@ -125,3 +128,26 @@
     @include rem(font-size, scale(4));
   }
 }
+
+.content-header__download_link {
+  @include inset(0, inline-end);
```

Comment:
Are both `float()` and `inset()` needed?

Reply:
No. Needs `inset()`, nuking `float()`.